### PR TITLE
feat: generation of an alcohols list based on the strength selected b…

### PR DESCRIPTION
**Pull request by jaalazard**
…y the user

## Description

Please include a summary of the change or which issue is fixed.

## Type of change

- [x ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Area of change

- [] Frontend
- [ x] Backend
- [ ] Other: .....

## General checklist:

- [ x] My code follows the style guidelines of this project
- [ x] I ran `bun run format`/`bun run lint` before commit
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] I have made corresponding changes to the documentation where needed
- [ x] My changes generate no new warnings
- [ x] I have checked my code and corrected any misspellings
- [x ] I have updated `develop` and merged to my branch before submitting pull request
- [ x] My pull request generate no conflicts with `develop` branch
- [ x] I requested code review from other team members

## Frontend checklist:

- [] I followed guidelines for `HTML`/`JSX`/`Typescript`
- [] My `Javascript` generate no new console errors
- [] I tested my code cross browsers
- [] My slice is pixel perfect for both desktop and mobile according to design
- [] I conducted basic QA to assure all features are working
- [] I tested responsive for mobile and tablet resolutions

## Backend checklist:

- [ x] I tested admin by manually adding content from zero
- [ x] I created easy to use admin experience which is self-explanatory
- [ x] I added description to admin fields in hard-to-understand areas
- [x ] I followed guidelines for naming `Typescript` variables
- [ x] I conducted basic QA to assure all features are working

### CODE REVIEW

**Comment by Cardamced** [comment]
T'as un souci dans le frontend, apparemment.

**Review by No3ml3 — DISMISSED** [review]

**Comment by No3ml3** [comment]
il (te) manque des cases mais sinon c'est okay

**Review by Cardamced — DISMISSED** [review]

File: packages/backend/api/src/router.ts [hunk 1]
```diff
@@ -24,4 +26,15 @@ router.get('/some-route', (_request, response) => {
   return response.json(value);
 });
 
+router.get('/alcohols/:level', async (req, res) => {
+  const degree = Number.parseInt(req.params.level);
```
Comment (YDrogen) [inline packages/backend/api/src/router.ts:30]: Pourquoi `Number.` ?

File: packages/backend/api/src/services/alcohol-service.ts [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import { type Kysely, sql } from 'kysely';
+
+import type { Database } from '@app/types';
+import type { Ingredient } from '@app/types';
```
Comment (YDrogen) [inline packages/backend/api/src/services/alcohol-service.ts]: Une seule ligne possible

File: packages/backend/api/src/services/alcohol-service.ts
Comment (YDrogen) [inline packages/backend/api/src/services/alcohol-service.ts:1]: Une raison pour ne pas utiliser le query builder [(doc)](https://kysely.dev/docs/examples/WHERE/conditional-where-calls) ?

**Review by No3ml3 — DISMISSED** [review]

**Review by Cardamced — DISMISSED** [review]

File: packages/frontend/web/src/pages/AddCocktail.tsx [hunk 3]
```diff
@@ -38,12 +41,24 @@ export default function AddCocktail() {
...
-  const handleClick = (number: number) => {
-    if (number === level) {
-      setLevel(0);
-    } else {
-      setLevel(number);
-      setValue('level', number);
+  const handleLevelClick = async (selectedLevel: number) => {
+    try {
+      const response = await fetch(
+        `http://localhost:3333/api/alcohols/${selectedLevel}`,
```
Comment (YDrogen) [inline packages/frontend/web/src/pages/AddCocktail.tsx]: Une pensée pour les copains qui sont pas sur `localhost` 🙏 
L'URL du backend se trouve dans les variables d'environnement, il y a un exemple sur la page Home

**Review by YDrogen — CHANGES_REQUESTED** [review]

File: packages/backend/api/src/router.ts (hunk 1 above)
Comment (jaalazard) [reply packages/backend/api/src/router.ts:30]: C'est ESLint qui m'a forcé moi je ne voulais pas

File: packages/backend/api/src/services/alcohol-service.ts
Comment (jaalazard) [reply packages/backend/api/src/services/alcohol-service.ts:1]: C'était une option non ?

**Review by Cardamced — DISMISSED** [review]
Ok pour moi.

**Review by No3ml3 — DISMISSED** [review]

**Review by Justicesvr — DISMISSED** [review]

**Review by YDrogen — APPROVED** [review]

**Review by YDrogen — CHANGES_REQUESTED** [review]
conflit

**Review by No3ml3 — APPROVED** [review]

**Review by Cardamced — APPROVED** [review]

**Review by YDrogen — APPROVED** [review]

**Pull request merged**